template/.npmignore.temp

```diff
@@ -1,16 +1,23 @@
 # standard ignore
+*config*
+*conf*
+
+.github/
 .gitignore
+.eslintrc
 .stylelintrc
-karma.conf.js
-rollup.config.js
-babel.config.js
-.github/
-index.html
+
+coverage/
```

Comment:
Shouldn't this be be filtered out of every commit and never end up as part of the NPM package anyhow?

Reply:
It is in the .gitignore and the .npmignore. I am not sure I understand the question?

Reply:
I mean - since it's never going to get into a commit due to .gitignore doesn't that already in itself filter it out of the npm package? Why do we need to try and filter it out a second time?

Reply:
No, if you remove this from the `.npmignore` this will get included with the package. Just validated. 
<img width="720" alt="Screen Shot 2021-05-10 at 1 03 31 PM" src="https://user-images.githubusercontent.com/181089/117717821-29d27f00-b190-11eb-8b0b-9930891efd4e.png">